src/components/LiquidityMining/Create/Steps/PairAndReward/index.tsx

```diff
@@ -1,101 +1,223 @@
 import { Pair, Token, TokenAmount } from '@swapr/sdk'
-import React, { useCallback, useState } from 'react'
-import { Box, Flex } from 'rebass'
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { Flex } from 'rebass'
 import styled from 'styled-components'
+import { CampaignType } from '../../../../../pages/LiquidityMining/Create'
+import { tryParseAmount } from '../../../../../state/swap/hooks'
+import { TYPE } from '../../../../../theme'
+import { unwrappedToken } from '../../../../../utils/wrappedCurrency'
+import NumericalInput from '../../../../Input/NumericalInput'
 import { CurrencySearchModal } from '../../../../SearchModal/CurrencySearchModal'
+
 import { PairSearchModal } from '../../../../SearchModal/PairSearchModal'
-import { Card, Divider } from '../../../styleds'
+import { SmoothGradientCard } from '../../../styleds'
 import AssetSelector from './AssetSelector'
 
 const FlexContainer = styled(Flex)`
+  justify-content: stretch;
+  width: 100%;
+  gap: 28px;
   ${props => props.theme.mediaWidth.upToExtraSmall`
     flex-direction: column;
   `}
 `
-
-const RewardTokenContainer = styled(Box)`
-  ${props => props.theme.mediaWidth.upToExtraSmall`
-    margin-top: 16px !important;
-  `}
+const AmountFlex = styled(Flex)`
+  width: max-content;
+  align-self: center;
+  border-radius: 4px;
+  background: rgba(0, 0, 0, 0.65);
+  backdrop-filter: blur(20px);
+  padding: 12px;
+`
+const StyledUnlimitedText = styled(TYPE.largeHeader)<{ active: boolean }>`
+  text-decoration: ${props => (props.active ? 'underline' : 'none')};
+  opacity: ${({ active }) => !active && '0.7'};
+  text-underline-offset: 6px;
+  line-height: 22px;
+  color: ${props => (props.active ? props.theme.text2 : props.theme.dark4)};
```

Comment:
Sometimes we are destructing props, others we don't. Let's keep consistency and always destruct props. This comment applies to every component.